hazelcast/src/main/java/com/hazelcast/config/AdvancedNetworkConfig.java

```diff
@@ -0,0 +1,401 @@
+        public NetworkConfig setOutboundPorts(Collection<Integer> outboundPorts) {
+            throw new UnsupportedOperationException();
+        }
+
+        @Override
+        public NetworkConfig addOutboundPort(int port) {
+            throw new UnsupportedOperationException();
+        }
+
+        @Override
+        public InterfacesConfig getInterfaces() {
+            return getMemberEndpoint().getInterfaces();
+        }
+
+        @Override
+        public NetworkConfig setInterfaces(InterfacesConfig interfaces) {
+            throw new UnsupportedOperationException();
+        }
+
+        @Override
+        public JoinConfig getJoin() {
+            return config.getJoin();
+        }
+
+        @Override
+        public NetworkConfig setJoin(JoinConfig join) {
+            throw new UnsupportedOperationException();
+        }
+
+        @Override
+        public String getPublicAddress() {
+            return getMemberEndpoint().getPublicAddress();
+        }
+
+        @Override
+        public NetworkConfig setPublicAddress(String publicAddress) {
+            throw new UnsupportedOperationException();
+        }
+
+        @Override
+        public SocketInterceptorConfig getSocketInterceptorConfig() {
+            return getMemberEndpoint().getSocketInterceptorConfig();
+        }
+
+        @Override
+        public NetworkConfig setSocketInterceptorConfig(SocketInterceptorConfig socketInterceptorConfig) {
+            throw new UnsupportedOperationException();
+        }
+
+        @Override
+        public SymmetricEncryptionConfig getSymmetricEncryptionConfig() {
+            return getMemberEndpoint().getSymmetricEncryptionConfig();
+        }
+
+        @Override
+        public NetworkConfig setSymmetricEncryptionConfig(SymmetricEncryptionConfig symmetricEncryptionConfig) {
+            throw new UnsupportedOperationException();
+        }
+
+        @Override
+        public SSLConfig getSSLConfig() {
+            return getMemberEndpoint().getSSLConfig();
+        }
+
+        @Override
+        public NetworkConfig setSSLConfig(SSLConfig sslConfig) {
+            throw new UnsupportedOperationException();
+        }
+
+        @Override
+        public MemberAddressProviderConfig getMemberAddressProviderConfig() {
+            return config.getMemberAddressProviderConfig();
+        }
+
+        @Override
+        public NetworkConfig setMemberAddressProviderConfig(MemberAddressProviderConfig memberAddressProviderConfig) {
+            throw new UnsupportedOperationException();
+        }
+
+        @Override
+        public NetworkConfig setIcmpFailureDetectorConfig(IcmpFailureDetectorConfig icmpFailureDetectorConfig) {
+            throw new UnsupportedOperationException();
+        }
+
+        @Override
+        public IcmpFailureDetectorConfig getIcmpFailureDetectorConfig() {
+            return config.getIcmpFailureDetectorConfig();
+        }
+
+        @Override
+        public String toString() {
```

Comment:
Looks like this `toString` is not working properly, super's fields are not in sync with the this's config.